client/components/PaymentProcessor.js

```diff
@@ -16,6 +16,14 @@ const PaymentProcessor = ({navigation, route}) => {
   const {paymentUrl} = route.params;
   const [hasErrored, setHasErrored] = useState(false);
 
+  const PreFetchOptions = {
```

Comment:
minor nitpick: this project uses camel case so this should be:
```suggestion
const preFetchOptions = {
```